lib/utils.js

```diff
@@ -195,3 +204,12 @@ var utils = module.exports = {
     return obj;
   },
 };
+
+function emitStripeWarning(warning) {
+  if (typeof process.emitWarning !== 'function') {
+    /* eslint-disable no-console */
```

Comment:
Can you just make sure this lint doesn't need to be re-enabled after the console call? (I'm honestly not sure.) It might be better just to use the single line disable `eslint-disable-line` like we had above.